src/global/components/input.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="input--container">
```

Comment:
Czy to aby najlepsza klasa, robimy to  według BEM? Jeżeli tak, to dwa myślniki sugerują że jest to modyfikator klasy input. Mamy bazowa klasę input i modyfikujemy ją na potrzeby container, czy to się zgadza? Jesli nie to musimy ją zmienić.

Reply:
Ta klasa w ogóle jest zbędna -> wyrzuciłem ją.